### PR TITLE
pages: add rollbacks page

### PR DESCRIPTION
**Pull request by lukeed**
/cc @nprogers @Albertozhao @signalnerve I'm not sure if you wanted to include info about pausing automatic deployments? If so, I'm not sure of how that works / how automatic deployments can be toggled on/off

### CODE REVIEW

File: products/pages/src/content/platform/rollbacks.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Rollbacks
+
+Rollbacks allow you to instantly revert your project to a previous production deployment.
+
+Any production deployment that has been successfully built is a valid rollback target. When your project has rolled back to a previous deployment, you may still rollback to deployments that are _newer_ than your current version. Please note that preview deployments are not valid rollback targets.
```
Comment (kristianfreeman) [inline products/pages/src/content/platform/rollbacks.md:5]: > When your project has rolled back to a previous deployment, you may still rollback to deployments that are _newer_ than your current version.

File: products/pages/src/content/platform/rollbacks.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+In order to perform a rollback, select a Pages project and ensure its Deployments tab is activated. Then browse the **All deployments** list and click on the actions menu for the desired target. A confirmation window will appear after clicking on the "Rollback to this deployment" option. When confirmed, your project's production deployment will change instantly.
```
Comment (kristianfreeman) [inline products/pages/src/content/platform/rollbacks.md:7]: is it instant or does it need to do a new deploy? that's neat if it is instant

**Review by kristianfreeman — APPROVED** [review]

File: products/pages/src/content/platform/rollbacks.md (hunk 2 above)
Comment (lukeed) [reply products/pages/src/content/platform/rollbacks.md:7]: It's instant – or at least near-instant :D Testing it out showed that the rollback happened in the time it took me to refresh the page

**Comment by rita3ko** [comment]
cc @nevikashah

**Comment by nevikashah** [comment]
I think we should leave pausing automatic deployments out of the rollbacks section of docs. But I do think we could add a  how to pause deployments in the "Getting Started" section. I can create a separate issue for that though. Let me know your thoughts.

**Pull request merged**